pkg/apis/picchu/v1alpha1/revision_types.go

```diff
@@ -24,6 +24,10 @@ type Istio struct {
 	TrafficPolicy *istio.TrafficPolicy `json:"trafficPolicy,omitempty"`
```

Comment:
```suggestion
	TrafficPolicy *istio.TrafficPolicy `json:"trafficPolicy,omitempty"`
	Sidecar       *IstioSidecar        `json:"sidecar,omitempty"`
```

let’s keep the Istio configuration together